rxdogtag/src/main/java/com/uber/rxdogtag/ObserverHandler.java

```diff
@@ -45,15 +43,6 @@
  */
 public interface ObserverHandler {
 
-  /**
-   * @return a list of ignorable packages. Useful if decorating observers that you know can be
-   *     safely ignored when deducing a target subscribe() point. It's recommended that classes in
-   *     these packages have their names kept in Proguard/R8 obfuscation as well.
-   */
-  default Collection<String> ignorablePackagePrefixes() {
```

Comment:
I don't follow - this is part of the builder now

Reply:
What I mean is that previously `ObserverHandler`'s had a standard way of declaring packages that should be ignored by RxDogTag. Now, they explicitly have to write a small method (the `configureWith`) instead of overriding a default interface method to add a package to be ignored. 

If most `ObserverHandler`'s are going to be declaring packages to be ignored, I feel it would be better to have a standardized interface (similar to the current one with `configureWith`).

Reply:
I don't think that's a good idea, as it's sort of stuffing larger scope configuration into an API that's supposed to just be a component of that configuration. The idea is that no `ObserverHandler` declares packages to be ignored because it's unrelated to their purpose.

I get what you're saying though, and agree it could be better. We could add something to the builder like `configureWith(Configurer configurer)` where `Configurer` is a SAM interface like this

```java
interface Configurer {
  void apply(Builder builder);
}
```

Then libraries could have some entry point like 

```java
public final class AutoDisposeRdt {
  private AutoDisposeRdt() {

  }

  public static void configure(Builder builder) {
    // ...
  }
}
```

Then finally used like this:

```java
RxDogTag.builder()
    .configureWith(AutoDisposeRdt::configure)
    .install()
```

Implemented in 2cb2a9f

Reply:
Yup, that looks good!